tests/libc/enc/tests.c.in

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This is a template used by CMake.
+
+extern int run_test(
+    const char* name,
+    int (*main)(int argc, const char* argv[]));
+
+int run_tests(void)
+{
+    int ret = 0;
+@run_test@
```

Comment:
If it is just the "test_name" that is changing between tests, it would be simpler to just pass along a 
-DTEST_NAME=abc
as a command line option

Reply:
It is not; it's an accumulation of code, two lines per test that all go into one file. You'd quickly exceed command line character limits.

Reply:
For kicks, `tests.c` ends up with 45098 characters over 434 lines in place of `@run_test@`, which actually might fit on the command line, but doesn't seem worth it.